### PR TITLE
JNG-4891 SDK : Automapped primitives transfer tests

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4891" title="JNG-4891" target="_blank">TaskJNG-4891</a>  SDK : Automapped transfer tests
  </td></table>
  <br />
 

<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4897" title="JNG-4897" target="_blank">TaskJNG-4897</a>  Primitive type automapped entity members test
  </td>
  <br />
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4894" title="JNG-4894" target="_blank">TaskJNG-4894</a>   Test the not valid default values field on an Entity
  </td></table>

### CODE REVIEW

**Review by robertcsakany — APPROVED** [review]

File: judo-runtime-core-jsl-itest/src/test/resources/model/PrimitivesModel/Primitives.jsl [hunk 1]
```diff
@@ -110,3 +110,25 @@ entity DefaultRequiredEntity {
...
+        field Scaled scaledAttrWithWrongPrecision = 123456789.34;
+}
+
+entity EntityWithNotValidScaleDefaultValue {
+        field Scaled scaledAttrWithWrongScale = 2.3466;
+}
+
+// entity EntityWithNotValidStringDefaultValue {
+//         field SmallString stringAttr = "To large string literal. To large string literal.To large string literal.To large string literal.To large string literal.Too larg";
+// }
```
Comment (bencegelei) [inline judo-runtime-core-jsl-itest/src/test/resources/model/PrimitivesModel/Primitives.jsl:131]: If this is commented due to a bug, please add ticket reference.

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/DefaultValueValidationTest.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+    EntityWithNotValidIntegerDefaultValueDao entityWithNotValidIntegerDefaultValueDao;
+
+    @Inject
+    EntityWithNotValidPrecisionDefaultValueDao entityWithNotValidPrecisionDefaultValueDao;
+
+    @Inject
+    EntityWithNotValidScaleDefaultValueDao entityWithNotValidScaleDefaultValueDao;
+
+//    @Inject
+//    EntityWithNotValidStringDefaultValueDao entityWithNotValidStringDefaultValueDao;
```
Comment (bencegelei) [inline judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/DefaultValueValidationTest.java:58]: If this is commented due to a bug, please add ticket reference.

**Review by bencegelei — CHANGES_REQUESTED** [review]

File: judo-runtime-core-jsl-itest/src/test/resources/model/PrimitivesModel/Primitives.jsl (hunk 1 above)
Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/src/test/resources/model/PrimitivesModel/Primitives.jsl:131]: [bdfe200](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/191/commits/bdfe200f6a3608a459951711ccba6415c3ec1996)

File: judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/DefaultValueValidationTest.java (hunk 2 above)
Comment (gaborflorian) [reply judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/DefaultValueValidationTest.java:58]: [bdfe200](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/191/commits/bdfe200f6a3608a459951711ccba6415c3ec1996)

**Review by Madmat8 — APPROVED** [review]

**Review by bencegelei — APPROVED** [review]

**Pull request merged**